components/pedigree/resources/src/main/resources/pedigree/model/dynamicGraph.js

```diff
@@ -689,26 +695,7 @@ define([
             if (!this.isPerson(v))
                 throw "Assertion failed: hasRelationships() is applied to a non-person";
 
-            return (this.DG.GG.v[v].length > 0); // if it had relationships it must have been alive at some point
-        },
-
-        getPossibleGenders: function( v )
-        {
-            // returns: - any gender if no partners or all partners are of unknown genders;
-            //          - opposite of the partner gender if partner genders do not conflict
-            //          - "U" if has partners of different genders (for now this is suported)
-            var possible = {"M": true, "F": true, "O": true, "U": true};
-
-            var partners = this.DG.GG.getAllPartners(v);
-
-            for (var i = 0; i < partners.length; i++) {
-                var partnerGender = this.getGender(partners[i]);
-                if (partnerGender != "U" && partnerGender != "O") {
-                    possible[partnerGender] = false;
-                }
-            }
-            //console.log("Possible genders for " + v + ": " + Helpers.stringifyObject(possible));
-            return possible;
+            return (this.DG.GG.v[v].length > 0);
```

Comment:
Agree. This commit only removed a misleading comment, and a larger refactoring should be done as part of some other commit